cot/src/admin.rs

```diff
@@ -67,10 +69,7 @@ async fn index(
         urls: &urls,
         model_managers: managers,
     };
-    Ok(Response::new_html(
-        StatusCode::OK,
-        Body::fixed(template.render()?),
-    ))
+    Ok(Html::new(template.render()?))
```

Comment:
I'm wondering if it makes sense to get rid of the `new` constructor and just allow writing this like `Ok(Html(sth))`. This would be potentially more consistent with `Json(something)`.

What do you think?

Reply:
Yes and no. The important distinction here is that for now, `Json` is used only as an extractor, and `Html` only as a response. As a result, the logic for ensuring a valid `Json` struct is within `FromRequest`, while `Html` can be created whenever by the user and it needs proper constructors.

Reply:
> needs proper constructors

Why? Its point is to mark some content as "being HTML"; it doesn't do any kind of validation inside.

Reply:
That makes sense now. I agree that should be done; however, I won't have time for this in the next few days and it's not that major to be blocking this PR. I created an [item in our project](https://github.com/orgs/cot-rs/projects/2/views/7?pane=issue&itemId=108354805) for this to tackle it before the next release, but I think we can finally merge this PR as is.